src/index.html

```diff
@@ -6,9 +6,27 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="home_page"><img src="images/logo.png" alt="home_page"></a>
+      <nav>
+        <ul>
+          <li class="nav_link nav_link1"><a href="index.html" class="is-active">Apple</a></li>
+          <li class="nav_link" >Samsung</li>
```

Comment:
All of the nav__link should be with double underline in class naming for BEM element and have <a> inside